maplibre/src/map_schedule.rs

```diff
@@ -43,13 +43,17 @@ impl<E: Environment> InteractiveMapSchedule<E> {
         wgpu_settings: WgpuSettings,
         renderer_settings: RendererSettings,
     ) -> Self {
+        let maxzoom = style.maxzoom.unwrap_or_default();
+        let minzoom = style.minzoom.unwrap_or_default();
+        // if maxzoom < zoom { error("mapstype error: maxzoom must be > zoom"); }
+        // if minzoom > zoom { error("mapstype error: minzoom must be < zoom"); }
```

Comment:
This brings up a great question: Where do we validate the style? There are probably more "invariants" we could check.
It should not happen in this file, because then we would need to copy&paste this code to the headless.rs file.

Ideally, it should be impossible to construct an invalid Style in the first place. Maybe we can add a constructor to the Style and validate it there.

For now it makes sense to a add a `self.validate()` function to the Style. We can call this then from anywhere.

Reply:
I started, but far from perfect. Update soon